external-crates/move/crates/move-compiler/tests/move_check/parser/function_incomplete.exp

```diff
@@ -1,9 +1,44 @@
 error[E01002]: unexpected token
-  ┌─ tests/move_check/parser/function_incomplete.move:5:1
+  ┌─ tests/move_check/parser/function_incomplete.move:6:5
   │
-5 │ 
-  │ ^
-  │ 
-  │ Unexpected end-of-file
-  │ Expected an expression term
+6 │     fun just_type_args<T>
+  │     ^^^
+  │     │
+  │     Unexpected 'fun'
+  │     Expected '('
+
+error[E01002]: unexpected token
+  ┌─ tests/move_check/parser/function_incomplete.move:8:5
+  │
+8 │     fun just_param<T>(_u: u64)
+  │     ^^^
+  │     │
+  │     Unexpected 'fun'
+  │     Expected '('
```

Comment:
Something the old version of the code did that my prototype did not is point to the correct location for these errors. For these, we should point out that we expected `(` at the previous declaration / end of the last token, and we found `fun` here.